### ISSUE TITLE
Не проверяется существование файла

### ISSUE THREAD

**Issue by Saniek1** [snewer/yii2-images#8]
Не проверяется существование файла при создании превью, из-за этого страница падает с ошибкой

**Comment by snewer** [comment]
@Saniek1 а куда файл делся? Удалили вручную?

**Comment by Saniek1** [comment]
Да, файла нет, т.к. проект только развернут, а БД уже заполнена

**Comment by snewer** [comment]
> Да, файла нет, т.к. проект только развернут, а БД уже заполнена

Не понятно чем заполнена бд.
Напишите, пожалуйста, мне в личку (телеграм @snewer), разберем проблему.

**Comment by VikingBO** [comment]
Та же проблема, проект развёрнут на локалке, БД накатана с дампа, файлов для миниатюр ещё нет, но пути сохранены в БД.

При попытке открыть страницу, где выводятся миниатюры, страница не открывается из-за exception потому как `file_get_content` по адресу `vendor/snewer/yii2-storage/src/drivers/FileSystemDriver.php:74` не может ничего получить.
